src/main/java/org/eurekaclinical/useragreement/webapp/client/EurekaClinicalUserAgreementInternalClient.java

```diff
@@ -0,0 +1,63 @@
+package org.eurekaclinical.useragreement.webapp.client;
+
```

Comment:
Looks fine, except delete the license header and then build to let it get recreated automatically with the right project name.